src/commands/add-approuter.ts

```diff
@@ -17,24 +17,32 @@ export default class AddApprouter extends Command {
   static flags = {
     projectDir: flags.string({
       hidden: true,
-      default: '',
-      description: 'Path to the folder in which the project should be created.'
+      description: 'Path to the project directory to which the approuter should be added.'
     }),
     force: flags.boolean({
       description: 'Do not fail if a file already exist and overwrite it.'
     }),
     help: flags.help({ char: 'h' })
   };
 
+  static args = [
+    {
+      name: 'projectDir',
```

Comment:
But is it an issue? I would argue either we have it or we don't. So either remove the flag or keep both for consistency (and make it non-hidden, I guess). What do you think?

Reply:
My assumption was that a simpler (public) API makes it easier to understand the docs. So omitting a flag/arg that is not useful for 99% of users (and can be simply worked around by `cd projectDir`) would be a positive change.

But I have not validated that assumption, so feel free to change my mind on this.

Reply:
I have removed all the `--projectDir` flags as discussed, but I decided to leave the argument, because I think it makes our commands consistent. I don't think `projectDir` makes the API hard to understand, it is still very simple, while the change is tiny.

Reply:
To throw my 2cts in, I also have no idea whether we need it or not. So the question is how do find that out in the future? Otherwise we'll have the same discussion on the next PR.